rs/nns/integration_tests/src/cycles_minting_canister_with_exchange_rate_canister.rs

```diff
@@ -1,21 +1,28 @@
-use candid::Encode;
+use candid::{CandidType, Encode};
 use canister_test::Wasm;
+use cycles_minting_canister::IcpXdrConversionRateCertifiedResponse;
 use ic_base_types::CanisterId;
+use ic_cbor::CertificateToCbor;
+use ic_certificate_verification::VerifyCertificate;
```

Comment:
Please use [ic-signature-verification](https://github.com/dfinity/ic/tree/master/packages/ic-signature-verification) instead of `ic-certificate-verification`. We will soon deprecate `ic-certificate-verification`.

Reply:
Discussed offline - the [ic-signature-verification](https://github.com/dfinity/ic/tree/master/packages/ic-signature-verification) crate cannot be used just yet because it has assumptions of using the canister signature scheme (check_sig_path).